snuba/util.py

```diff
@@ -207,24 +208,24 @@ def is_condition(cond_or_list):
     )
 
 
-def all_referenced_columns(body):
+def all_referenced_columns(query):
```

Comment:
Was there a specific reason to pass query to this function instead of request as it happens to most the functions you updated?
I think it would be easier to understand for the reader if this received a request and then extracted the query from there. It would give more context.

Reply:
It didn't require the remainder of the request data (extensions, etc.)

My suspicion is that later the request will have the extensions applied to it to return a new query (`f(Request) -> Query` somewhere) which will be what actually gets passed through everything related to query execution, but I think that will be dependent on the dataset changes mentioned above too.